xbee_tx_block.py

```diff
@@ -13,23 +13,32 @@ class XBeeTX(XBeeBase):
     then the block will notify the signal.
 
     Parameters:
-        dest_addr: 2 byte address of remote xbee to send AT command too.
-            Default value when left blank is "FF FF" which sends a broadcast.
+        dest_addr: address of remote xbee to send AT command too.
+            Default value when left blank is "FF FF" which sends a broadcast
+            or the "Destination Address" configured in the XBee
     """
 
     version = VersionProperty(version='0.2.1')
     data = Property(title="Data", default="{{ $.to_dict() }}")
+    # todo: make dest_addr a block property
 
     def process_signals(self, signals):
         for signal in signals:
             data_encoded = "{}".format(self.data(signal)).encode()
             self.logger.debug('Sending data: {}'.format(data_encoded))
-            # tx: 0x01 "Tx (Transmit) Request: 16-bit address"
```

Comment:
I think these comments are pretty useful. Can you add them back in? And for all the non-digimesh ones we don't need to pass all the extra params. I thin it would be better not to. The only reason we have to do it in digimesh is for the bug in the python library.